requirements/requirements-dev.txt

```diff
@@ -3,6 +3,7 @@ clang-format==18.1.3
 comet_ml>=3.41.0
 deepspeed-kernels ; sys_platform == 'linux'
 docutils<0.18
+einops
```

Comment:
The other PR has merged so you should be free to remove this now.

Reply:
Done.

Reply:
Thanks!